### PR TITLE
Fix/design review author profile pagination

### PR DESCRIPTION
**Pull request by rogeriopvl**
Here's a list of the fixes in this PR:

- Change margin between top of pagination and buff background to 30px
- Increase height of top and bottom pagination to 50px
- 'Previous' should say 'Previous page' and 'Next' should say 'Next page'
- Change next & previous chevron dimensions to: Width 7px, Height 12px
- Widen pagination keylines so that they reach the edges of the screen on mobile
- Previous and Next should retain their existing wording (‘page’ is only added on desktop and tablet. It’s removed on mobile to save space.)
- Remove top keyline from pagination at the top of the page

And here's a screenshot:

image

### CODE REVIEW

File: packages/author-profile/author-profile-content.web.js [hunk 1]
```diff
@@ -25,11 +25,11 @@ const ContentContainer = withResponsiveStyles(View, {
     width: 100%;
     max-width: 680px;
     padding-left: 10px;
-    padding-right 10px;
+    padding-right: 10px;
   `,
   mediumUp: () => `
     padding-left: 0;
-    padding-right 0;
+    padding-right: 0;
```
Comment (colincclark) [inline packages/author-profile/author-profile-content.web.js]: 👍

File: packages/author-profile/author-profile-content.web.js [hunk 2]
```diff
@@ -157,10 +157,14 @@ class AuthorProfileContent extends Component {
       refetch
     } = this.props;
 
-    const paginationComponent = (hideResults = false) => (
+    const paginationComponent = (
```
Comment (toboid) [inline packages/author-profile/author-profile-content.web.js]: How about using an options object? That way the calls will show what the args mean:
`paginationComponent(false, true)`
could be changed to
`paginationComponent({
  hideResults: false,
  hideTopKeyline: true
})`

Comment (rogeriopvl) [reply packages/author-profile/author-profile-content.web.js]: I agree 👍

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/2cc1f1bbbc994ee7

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/fd072def70eb6533

File: packages/pagination/pagination.js [hunk 3]
```diff
@@ -19,19 +19,25 @@ const styles = StyleSheet.create({
...
-    borderTopWidth: 1,
     alignItems: "center",
     flexDirection: "row",
     justifyContent: "space-between",
-    height: 40
+    height: 50
+  },
+  borderTop: {
+    borderTopColor: "#dbdbdb",
+    borderTopWidth: 1,
```
Comment (colincclark) [inline packages/pagination/pagination.js]: trailing comma

File: packages/pagination/pagination.js [hunk 4]
```diff
@@ -19,19 +19,25 @@ const styles = StyleSheet.create({
...
+  },
+  borderBottom: {
+    borderBottomColor: "#dbdbdb",
+    borderBottomWidth: 1,
```
Comment (colincclark) [inline packages/pagination/pagination.js]: trailing comma

**Comment by colincclark** [comment]
Running `yarn fmt` locally fixed the linting issues you are having

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/e9a0c0c404cd9d36

**Pull request merged**